runtime/js/90_deno_ns.js

```diff
@@ -129,6 +129,7 @@ const denoNs = {
   PermissionStatus: permissions.PermissionStatus,
   // TODO(bartlomieju): why is this not in one of extensions?
   serveHttp: httpRuntime.serveHttp,
+  serve: http.serve,
```

Comment:
Are there any tests we can update to not use `--unstable`?

Reply:
It appears, there's only a single one.